### PR TITLE
Improve README.md for recursor compilation

### PR DESCRIPTION
**Pull request by HLFH**
@pieterlexis Hi. I'm more a tennis guy.

### CODE REVIEW

**Comment by HLFH** [comment]
@pieterlexis I also updated recursing.md to make it work with IPv6.

File: docs/markdown/authoritative/recursion.md [hunk 1]
```diff
@@ -41,7 +41,22 @@ if the RD flag were unset and the answer will indicate that recursion was not
...
+> A host identified by an Internet Protocol literal address, version 6 [RFC3513] or
+later, is distinguished by enclosing the IP literal within square brackets ("[" and "]").
+This is the only place where square bracket characters are allowed in the URI syntax.
+In anticipation of future, as-yet-undefined IP literal address formats, an
+implementation may use an optional version flag to indicate such a format explicitly
+rather than rely on heuristic determination.
+
+So, be careful! `pdns-recursor` can't work in IPv6 if you write something wrongly
+like ~~`recursor=::1:5300`~~ without the required square brackets ("[" and "]") 
+enclosing the IP litteral.
```
Comment (pieterlexis) [inline docs/markdown/authoritative/recursion.md]: s/litteral/literal/

File: docs/markdown/authoritative/recursion.md [hunk 2]
```diff
@@ -41,7 +41,22 @@ if the RD flag were unset and the answer will indicate that recursion was not
...
+they explain in section 3.2.2: Host:
+
+> A host identified by an Internet Protocol literal address, version 6 [RFC3513] or
+later, is distinguished by enclosing the IP literal within square brackets ("[" and "]").
+This is the only place where square bracket characters are allowed in the URI syntax.
+In anticipation of future, as-yet-undefined IP literal address formats, an
+implementation may use an optional version flag to indicate such a format explicitly
+rather than rely on heuristic determination.
+
+So, be careful! `pdns-recursor` can't work in IPv6 if you write something wrongly
```
Comment (pieterlexis) [inline docs/markdown/authoritative/recursion.md]: this document has nothing to do with the pdns-recursor

**Comment by pieterlexis** [comment]
2 nits, lgtm otherwise

**Comment by HLFH** [comment]
@pieterlexis Thanks. I fixed the typo and explained the big issue we have with bad IPv6 notation. The bad IPv6 notation is in strikethrough text. Is that an issue? And is everything clear? https://github.com/HLFH/pdns/blob/master/docs/markdown/authoritative/recursion.md

**Comment by pieterlexis** [comment]
:+1:

**Pull request merged**